public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -808,7 +808,7 @@ function AddAnomalyDetector({
                         <EuiText size="s">
                           <p>
                             The dashboard does not support high-cardinality
-                            detectors.
+                            detectors. 
```

Comment:
Does this get automatically removed with `prettier` when it removes trailing whitespace? What about changing to `{' '}` or wrap the entire text in ` format quotes to ensure it can't be changed?

Reply:
yes it gets removed with yarn prettier. Changed it to a non-breaking space character to prevent this happen again.
![Screenshot 2023-06-27 at 12 02 57 PM](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/assets/41348518/fbd601b6-8c38-4ffd-afc0-bbb04d453837)